code/modules/mob/living/say.dm

```diff
@@ -37,36 +37,36 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	//kinda localization -- rastaf0
 	//same keys as above, but on russian keyboard layout. This file uses cp1251 as encoding.
 	// Location
-	"�" = "right hand",
-	"�" = "left hand",
-	"�" = "intercom",
+	"ê" = "right hand",
+	"ä" = "left hand",

```

Comment:
I'm pretty sure I didn't.

I broke the encoding and then fixed it and the diff on github is just flipping out.